app/Http/Controllers/Admin/CategoriesController.php

```diff
@@ -58,10 +58,15 @@ public function store(CategoryRequests $request)
     /**
      * Show the form for editing the specified resource.
      *
+     * @param Category $category category object
+     *
      * @return \Illuminate\Http\Response
     */
-    public function edit()
+    public function edit(Category $category)
```

Comment:
làm như vậy thì họ muốn update tên của category food và drink thì làm sao em ?

Reply:
à vấn đề đó nhóm em đã quyết định là không thay đổi và xóa food and drink hai cái food , drink là mặc định của web